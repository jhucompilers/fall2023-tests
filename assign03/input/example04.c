int sum(int arr[0], int n);

int main(void) {
  int a[5];
  a[0] = 1;
  a[1] = 2;
  a[2] = 3;
  a[3] = 4;
  a[4] = 5;
  int tot;
  tot = sum(a, 5);
  return tot;
}
