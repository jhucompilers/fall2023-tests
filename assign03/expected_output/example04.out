0|sum|function|function (array of 0 x int, int) returning int
0|main|function|function () returning int
2|a|variable|array of 5 x int
2|tot|variable|int
